patches/minecraft/net/minecraft/world/gen/feature/WorldGenerator.java.patch

```diff
@@ -0,0 +1,12 @@
+--- ../src-base/minecraft/net/minecraft/world/gen/feature/WorldGenerator.java
++++ ../src-work/minecraft/net/minecraft/world/gen/feature/WorldGenerator.java
+@@ -33,7 +33,8 @@
+         }
+         else
+         {
+-            p_175903_1_.func_180501_a(p_175903_2_, p_175903_3_, 2);
++            int flag = net.minecraftforge.common.ForgeModContainer.fixVanillaCascading ? 2| 16 : 2; //Forge: With bit 5 unset, it will notify neighbors and load adjacent chunks.
```

Comment:
If we're adding an extra line anyway, might as well put the comment on top for readability.

Reply:
I wanted to match how Lex did the exact same thing to [WorldGenLakes.java.patch](https://github.com/josephcsible/MinecraftForge/commit/0ad4218bc2d3a8cc13357d631fe1b2818e7a7e21#diff-65418d5bcae199ee6e3eba086ce3cc24).

Reply:
You could NOT add the second line.